appserver/packager/data/pom.xml

```diff
@@ -89,6 +89,16 @@
             <artifactId>jakarta.data-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>fish.payara.server.internal.data</groupId>
+            <artifactId>data-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.classgraph</groupId>
+            <artifactId>classgraph</artifactId>
+            <version>${classgraph.version}</version>
```

Comment:
This should be defined in dependency-management in the top-level or appserver parent pom